cocos/2d/CCSpriteFrameCache.cpp

```diff
@@ -142,7 +151,7 @@ void SpriteFrameCache::removeUnusedSpriteFrames()
     for (auto& iter : frames)
     {
         auto* spriteFrame = iter.second;
-        if (spriteFrame->getReferenceCount() == 1)
+        if (spriteFrame->getReferenceCount() <= 2) // A frame has a ref count of 1 when created, and 2 when added to cocos2d::Map. It is unused if the ref count is 2.
```

Comment:
Fix original frame create code maybe better

Reply:
The SpriteFrame::createWithTexture use autorelease, so it seems the referenceCount is correct and should be 1 when a spriteFrame unused, see:
https://github.com/adxeproject/adxe/blob/188345e42a68427e4ab347e3f7b4ba31dbedb007/cocos/2d/CCPlistSpriteSheetLoader.cpp#L224

Reply:
That is so strange. I'll keep looking into it